DeepSpeech.py

```diff
@@ -761,14 +761,17 @@ def calculate_report(results_tuple):
     samples = []
     items = list(zip(*results_tuple))
     mean_wer = 0.0
+    total_label_length = 0.0
     for label, decoding, distance, loss in items:
         sample_wer = wer(label, decoding)
         sample = Sample(label, decoding, loss, distance, sample_wer)
         samples.append(sample)
-        mean_wer += sample_wer
+        label_length = len(label.split())
+        mean_wer += sample_wer * label_length
```

Comment:
See above comment. You should also rename `mean_wer` it to something like `total_edit_distance`.